python/ray/plasma/format/plasma.fbs

```diff
@@ -0,0 +1,331 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This file is not needed, right? (same with `python/ray/plasma/format/__init__.py`)